infra/azure/main.tf

```diff
@@ -306,6 +306,12 @@ resource "azurerm_storage_container" "batch_logs" {
   container_access_type = "private"
 }
 
+resource "azurerm_storage_container" "query" {
+  name                  = "query"
+  storage_account_name  = azurerm_storage_account.batch.name
+  container_access_type = "private"
+}
```

Comment:
As is, only the batch service principal will have access to this container. Do any other service accounts need access?

Reply:
No, in the long run Batch workers will fetch the JAR.

Reply:
In that case you'll need to give the batch worker managed identity access to the container

Reply:
This is a new container, not a new account. AFAIK, the batch worker already has access to the batch storage account.

Reply:
Ah, you're right! Never mind.